src/vmm/src/cpu_config/x86_64/cpuid/normalize.rs

```diff
@@ -365,11 +360,19 @@ impl super::Cpuid {
                     }
                     // Core domain
                     1 => {
-                        set_range(&mut subleaf.result.eax, 0..5, LEAFBH_INDEX1_APICID)
-                            .map_err(ExtendedTopologyError::ApicId)?;
-
                         // Configure such that the next higher-scoped domain (i.e. socket) include
                         // all logical processors.
+                        //
+                        // The CPUID.(EAX=0BH,ECX=1).EAX[4:0] value should be greater than or equal
+                        // to the base 2 logarithm of the number of logical processors, rounded up
+                        // to the decimal point. Since the maximum number of vCPUs is hardcoded and
+                        // is known a factorial of 2, use ilog2() of it.
+                        set_range(
+                            &mut subleaf.result.eax,
+                            0..5,
+                            crate::vmm_config::machine_config::MAX_SUPPORTED_VCPUS.ilog2(),
```

Comment:
nit: If we add a `use` for this constant, this continues fitting onto one line :)